install.sh

```diff
@@ -28,5 +28,8 @@ BDM_PROJECT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 # include util functions
 . $BDM_PROJECT_DIR/util/installation/common/util.sh
 
+# Install all prerequisites
+$BDM_PROJECT_DIR/prerequisites.sh $(DetectOs) all
```

Comment:
Let's only detect the OS if the user didn't specify one.
Similar to the CMake solution